client/index.css

```diff
@@ -16,4 +18,56 @@
   padding: 1em;
   border: 1px solid #ddd;
   text-align: center;
+}
+
+p {
+  font-family: Goudy Old Style;
+  color: var(--main-txt-color);
+}
+
+th {
+  font-family: Goudy Old Style;
+  color: var(--main-txt-color);
+  border: 1px solid var(--main-txt-color);
+  padding: 10px;
+  padding-bottom: 1px;
+  padding-top: 1px;
+}
+
+h1 {
+  font-family: Goudy Old Style;
+  text-align: center;
+  color: var(--main-txt-color);
```

Comment:
We're repeating this a few times all over the place. Instead, can you just set this in the "body" css element, which all of these nerds inherit from. That way, by default all of these will have that too, and anyone that wants something different can explicitly override that with some other value.